setup.py

```diff
@@ -2,36 +2,59 @@
 
 from distutils.core import setup
 from distutils.extension import Extension
-from Cython.Distutils import build_ext
 
 # Complain on 32-bit systems. See README for more details
 import struct
 if struct.calcsize('P') < 8:
     raise RuntimeError(
         'Simhash-py does not work on 32-bit systems. See README.md')
 
+ext_files = [
+    'simhash/simhash-cpp/src/permutation.cpp',
+    'simhash/simhash-cpp/src/simhash.cpp'
+]
+
+kwargs = {}
 
-ext_modules = [Extension('simhash.table', [
-        'simhash/table.pyx',
-        'simhash/simhash-cpp/src/simhash.cpp'
-    ], language='c++', libraries=['Judy']),
+try:
+    from Cython.Distutils import build_ext
+    print 'Building from Cython'
+    ext_files.append('simhash/simhash.pyx')
+    kwargs['cmdclass'] = {'build_ext': build_ext}
+except ImportError:
+    print 'Buidling from C++'
+    ext_files.append('simhash/simhash.cpp')
+
+ext_modules = [
+    Extension('simhash.simhash', ext_files,
+        language='c++',
+        extra_compile_args=['-std=c++11'],
+        include_dirs=['simhash/simhash-cpp/include']
+    )
 ]
 
 setup(name           = 'simhash',
-    version          = '0.1.1',
+    version          = '0.2.0rc1',
     description      = 'Near-Duplicate Detection with Simhash',
     url              = 'http://github.com/seomoz/simhash-py',
     author           = 'Dan Lecocq',
     author_email     = 'dan@moz.com',
-    packages         = ['simhash'],
-    package_dir      = {'simhash': 'simhash'},
-    cmdclass         = {'build_ext': build_ext},
-    install_requires = [],
```

Comment:
I think that we decided that although it repeats itself, we need to include install dependencies here as well as in `requirements.txt` so that `python setup.py install` gets what it needs.